definitions/deep-diff_v0.3.x/flow_>=v0.23.x/deep-diff_v0.3.x.js

```diff
@@ -0,0 +1,18 @@
+type Difference = {
```

Comment:
Reading the docs -- it seems like this would be well-suited to be typed as a disjoint union (discriminated by the `kind` property) since `lhs` doesn't exist on `kind === "N"`, `rhs` doesn't exist on `kind === "D"`, `index` only exists when `kind === "A"`, etc..

```javascript
type Difference = {
};
```